lib/query/country_divisions.rb

```diff
@@ -41,7 +44,9 @@ def division_results
     end
 
     def legislatures(place_id)
-      division_results.select { |r| r[:item] == place_id }.map { |i| i[:legislature] }.uniq
+      results_for_place = division_results.select { |result| result[:item] == place_id }
+      place_results_with_active_legislatures = results_for_place.delete_if { |result| result[:legislatureEnd] }
+      place_results_with_active_legislatures.map { |result| result[:legislature] }.uniq
```

Comment:
This method was already a little suspect, but this is pushing it much further. That these intermediate variables can't themselves be methods without passing lots of variables around shows that all this logic is probably in the wrong place. (NB, `reek` is also raising two new errors on this. Even though we don't have that turned on explicitly, it's worth running it and at least leaving a comment about things you're not fixing.)